src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -2193,19 +2209,6 @@ internal void GenerateAnonymousFunctionConversionError(BindingDiagnosticBag diag
                         Error(diagnostics, ErrorCode.ERR_BadParamType, lambdaParameterLocation,
                             i + 1, lambdaRefKind.ToParameterPrefix(), distinguisher.First, delegateRefKind.ToParameterPrefix(), distinguisher.Second);
```

Comment:
It looks like we no longer have any tests where `lambdaRefKind` or `delegateRefKind` would be something else than `None` when this error is reported. Consider adding some tests.

Reply:
i don't understand this bit.  the refkinds can definitely be non-none.  For example, if you have `ref int` and `ref byte`.  They'll pass the ref test, then fail the type check here.

Reply:
Yes, but do we have tests demonstrating hitting that code path?

Reply:
Added explicit test of this case :)